src/cli/commands/install.js

```diff
@@ -628,12 +668,24 @@ export class Install {
     for (const step of steps) {
       const stepResult = await step(++currentStep, steps.length);
       if (stepResult && stepResult.bailout) {
+        if (this.flags.audit) {
```

Comment:
Not sure this works - when running `install --audit` twice, the second time I get no reports

Reply:
I think it's because the bailout is done at the end of the resolution step, before the audit gets initialized

Reply:
Yeah I'm pretty sure yarn just bails before it gets that far, like it would have during a normal `install`.

Reply:
I think it should be safe to disable the integrity check altogether when `--audit` is set.